consensus/polybft/validator_account.go

```diff
@@ -5,93 +5,110 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"math/big"
 	"reflect"
 
 	bls "github.com/0xPolygon/polygon-edge/consensus/polybft/signer"
 
-	"github.com/0xPolygon/pbft-consensus"
 	"github.com/0xPolygon/polygon-edge/consensus/polybft/bitmap"
 	"github.com/0xPolygon/polygon-edge/types"
 	"github.com/umbracle/fastrlp"
 )
 
-// ValidatorAccount represents a validator from the validator set
-type ValidatorAccount struct {
-	Address types.Address
-	BlsKey  *bls.PublicKey
+// ValidatorMetadata represents a validator metadata (its public identity)
+type ValidatorMetadata struct {
```

Comment:
filename refactor to `validator_metadata`?

Reply:
Fixed [3e02cbf](https://github.com/0xPolygon/polygon-edge/pull/865/commits/3e02cbfb20a8c1d93f82be235b8a8a751a36bdbc)